lib/build/helpers/TaskUtil.js

```diff
@@ -172,7 +182,8 @@ class TaskUtil {
 	 * This method is only available to custom task extensions defining
 	 * <b>Specification Version 2.2 and above</b>.
 	 *
-	 * @param {Function} callback Callback to register. If it returns a Promise, it will be waited for
+	 * @param {@ui5/project/build/helpers/TaskUtil~cleanupTaskCallback} callback Callback to register.
+	 * 			If it returns a Promise, it will be waited for
```

Comment:
```suggestion
	 * @param {@ui5/project/build/helpers/TaskUtil~cleanupTaskCallback} callback Callback to register; it will be waited for if it returns a Promise
	 * 			
```